<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="Student">
       <property name="text">
        <string>Начать работу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="List">
       <property name="text">
        <string>Просмотреть лист задания</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Programm">
       <property name="text">
        <string>Написать программу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Otchet">
       <property name="text">
        <string>Написать отчет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Question">
       <property name="text">
        <string>Ответить на контрольные вопросы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>881</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="PC">
       <property name="text">
        <string>Включить компьютер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PO">
       <property name="text">
        <string>Изучить предметную область</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Model">
       <property name="text">
        <string>Создать модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Info">
       <property name="text">
        <string>Получить дополнительную информацию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Word">
       <property name="text">
        <string>Открыть текстовый процессор</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="view_model">
       <property name="text">
        <string>Просмотреть модель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="links">
       <property name="text">
        <string>Просмотреть связи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="atr">
       <property name="text">
        <string>Просмотреть атрибуты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>Сброс</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1031</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
